website/app/controllers/show.js

```diff
@@ -147,6 +147,20 @@ export default class ShowController extends Controller {
 
   @action
   setCurrent(current) {
+    // update the query params if tabs exist and current is defined
+    if (this.tabs.length > 1 && current !== null) {
+      if (current == 0) {
+        // for the first tab we remove the query param
+        set(this, 'selectedTab', null);
+      } else {
+        // for the rest of the tabs we set the query param tab to a lowercase version of the tab label
+        set(this, 'selectedTab', this.tabs[current].label.toLowerCase());
+      }
+    } else {
+      // make the first tab current if not defined
+      current = 0;
+    }
+
     // TABS
     // ?? CAN WE EXPLICITLY SET FOCUS ON THE TAB WHEN IT IS SET TO CURRENT?
```

Comment:
this is still relevant- if you click on a tab with your mouse, then press the TAB key, you'll start from the top of the page instead of the currently selected/focused tab.